pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -102,6 +102,10 @@ func (info *ClaimInfo) setCDIDevices(pluginName string, cdiDevices []string) err
 		info.CDIDevices = make(map[string][]string)
 	}
 
+	if info.annotations == nil {
+		info.annotations = make(map[string][]kubecontainer.Annotation)
+	}
```

Comment:
Looks like you found a bug when writing the unit tests?

Reply:
I thought you had added them for a reason when adding the unit test (it's in that commit). :shrug: